servicetest/networkgateway/test_network_gateway.py

```diff
@@ -110,13 +110,18 @@ def test_network_policy_drop(self):
         """
         try:
             sc = Container()
-            sc.start(DATA)
-            sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
-            sc.launch_command("python " +
+            success = sc.start(DATA)
+            assert success is True
+            
+            success = sc.set_gateway_config("network", GW_CONFIG_POLICY_DROP)
+            assert success is True
+
+            pid, success = sc.launch_command("python " +
                               sc.get_bind_dir() +
                               "/testhelper.py" +
                               " --test-dir " + sc.get_bind_dir() +
                               " --do-ping example.com")
+            assert success is True
```

Comment:
If it was possible, it would be nice to do all these asserts as part of the Container class in the test framework. In the service tests, things aren't usually supposed to fail, are they?

Or, we could just ignore the return value if we feel confident about the test setup. But I agree with that it looks bad having them clutter all tests.